common/tls.go

```diff
@@ -85,3 +89,45 @@ func RootCAs() *x509.CertPool {
 	}
 	return p
 }
+
+type TrustBundleResponse struct {
+	Certificate string `json:"certificate"`
+}
+
+// TrustBundle root CA certificates pool for connecting to EdgeHub Gateway.
+//
+// The TrustBundle is generated by the Edge Runtime at startup. The connection
+// to the edgeHub (proxy for IoT Hub) requires this cert. This CA cert can be
+// pulled from an API exposed on the edge runtime detailed in the below function.
+func TrustBundle(workloadUri, apiVersion string) *x509.CertPool {
```

Comment:
I think we shouldn't expose api version, because this is a part of the client's implementation and changing it may lead to undefined behavior.

https://github.com/amenzhinsky/iothub/blob/master/iotservice/client.go#L1336 `iotservice` has it's own API version, if it's common for all components we can extract it to the common package.